lib/module.js

```diff
@@ -5,7 +5,10 @@ module.exports = function () {
         enforce: 'pre',
         test: /\.(js|vue)$/,
         loader: 'eslint-loader',
-        exclude: /(node_modules)/
+        exclude: /(node_modules)/,
+        options: {
```

Comment:
We could improve this by just using `options`.
At the beginning of the module merge the options:
```js
module.exports = function (moduleOptions) {
  const options = {
    fix: false,
    ...this.options.eslint,
    ...moduleOptions
  }
```